src/tools/cephfs/MDSUtility.cc

```diff
@@ -108,6 +112,7 @@ int MDSUtility::init()
 
   finisher.start();
 
+  inited = true;
```

Comment:
@batrick I suspect these changes in src/tools/cephfs/MDSUtility.cc broke the jewel rados suite - see http://tracker.ceph.com/issues/23529

The entire api_tmap_migrate test was dropped in Kraken; I guess that's why master integration testing did not catch this.

Since this PR has already been merged into jewel, we would need to either revert or fix. Please advise?